### PR TITLE
Bncc 63

### PR DESCRIPTION
**Pull request by KepinTheNoob**

### CODE REVIEW

File: src/controllers/assignmentController.ts [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  successRes,
+} from "../utils";
+import { Assignment, Course } from "../models";
+import { ValidationError } from "sequelize";
+
+export const createAssignment: RequestHandler = async (req, res, next) => {
+  try {
+    const { course_id } = req.body;
+
+    const course = await Course.findOne({
```
Comment (gazmanzara) [inline src/controllers/assignmentController.ts:15]: pake findByPk aja

File: tsconfig.json [hunk 2]
```diff
@@ -8,6 +8,6 @@
     "skipLibCheck": true,
     "forceConsistentCasingInFileNames": true
   },
-  "include": ["**/*.ts"],
+  "include": ["**/*.ts", "src/config/config.ts"],
```
Comment (gazmanzara) [inline tsconfig.json:11]: balikin

File: src/controllers/assignmentController.ts [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+      },
+      {
+        where: {
+          id,
+        },
+      }
+    );
+
+    if (!data[0])
+      return errNotFound(next, `Assignment with id ${id} does not exist!`);
```
Comment (Specticall) [inline src/controllers/assignmentController.ts:95]: ini errornya jadiin bad request pake message `Failed to create assignment with the id ${id}`

File: src/controllers/assignmentController.ts [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+    return errInternalServer(next);
+  }
+};
+
+export const getAssignment: RequestHandler = async (req, res, next) => {
+  try {
+    const { id } = req.params;
+
+    const data = await Assignment.findOne({
```
Comment (gazmanzara) [inline src/controllers/assignmentController.ts:51]: findByPK

**Pull request merged**